### PR TITLE
Re-arrange and improve readability of product-tests README

### PR DESCRIPTION
**Pull request by petroav**

### CODE REVIEW

**Comment by petroav** [comment]
@fiedukow, @brian-rickman, @arhimondr: take a look when you get a chance. I re-structured the README a little. Let me know if you disagree with my changes.

**Comment by petroav** [comment]
Also, apologies for the spelling mistake in the commit message, I've fixed it locally.

**Comment by arhimondr** [comment]
@petroav Great job! Thanks.
I have rebased the `product-tests-intellij`, so you should probably rebase your PR on the actual branch.

#180 should fix the product tests failures.

**Comment by petroav** [comment]
Rebased. I'll wait for @fiedukow to take a look as well before merging into `product-tests-intellij`.

**Comment by petroav** [comment]
Added a `Troubleshooting` section at the end of the README. I'm merging into the `product-tests-intellij` branch.

File: presto-product-tests/README.md [hunk 1]
```diff
@@ -17,153 +17,231 @@
...
-If one does not want to use the ```default``` machine to run product tests, the following
- steps may be used to create and use a new machine named ```<machine>```:
+Alternatively, if you do not want to use the ```default``` VM to run the
+product tests, you can create a new VM with the commands below. Note that
+the ``default`` VM will always be running when you start a new pre-configured
+shell environment. Permanently removing or replacing the ``default`` VM
+is beyond the scope of this tutorial.
 
-* To create a virtual machine with name <machine> (required one time):
+* Create a VM called <machine>. This should be done only once and not
```
Comment (amrutagokhale) [inline presto-product-tests/README.md:57]: I guess <machine> should be surrounded with quotes so as to make it as ```<machine>```.

**Pull request closed without merging**